packages/dev/s2-docs/pages/react-aria/blog/how-we-internationalized-our-numberfield.mdx

```diff
@@ -0,0 +1,161 @@
+
+```tsx render
+import {NumberField} from '@react-spectrum/s2';
+
+<NumberField
+  label="Exposure"
+  formatOptions={{
+    signDisplay: 'exceptZero',
+    minimumFractionDigits: 1,
+    maximumFractionDigits: 2
+  }}
+  defaultValue={0}
+  step={0.1} />
+```
+
+### Allowed characters
+
+A number field should help the user enter a valid number and avoid accidental input. In order to do this, we only allow the user to type characters that meet the formatting requirements. For example, when the number is a percentage, we allow only numerals and the percent sign, and all other characters are ignored. The allowed characters are based on the formatting options as well as the current locale.
+
+In addition, we also support several different numbering systems, including the Latin, Arabic, and Han positional decimal systems. There are many different ways of entering numbers, including different hardware keyboard layouts, and various input method editors such as Pinyin, which uses combinations of Latin characters to represent Chinese logograms. These are supported via composition events, which the browser fires as the user enters each Latin character. While these characters by themselves are not valid numbers, we cannot validate them until the sequence is complete and the Latin characters are replaced by the Chinese logogram.
+
+<figure style={{display: 'flex', flexDirection: 'column', alignItems: 'center', margin: '20px 0'}}>
+  <video src={pinyinEntering} loop autoPlay muted style={{margin: 0, marginBottom: 5, width: '300px'}} />
+  <figcaption>Entering the number 21 in the Han positional decimal system using the iOS Pinyin keyboard</figcaption>
+</figure>
+
+### Mobile
+
+On mobile devices, the keyboard should be as helpful as possible, providing only the characters that can be entered into the field. This can vary based on the formatting options provided to the number field. For example, if the minimum value is greater than zero, no minus key should be displayed, and if fractional values are not allowed, then no decimal point should appear.
+
+The `inputMode` DOM attribute can be used to control the on screen keyboard shown by mobile devices. However, standard numeric keyboards vary across devices and operating systems making it difficult to provide a unified experience. For example, the iOS numeric keyboard does not include a minus sign at all, which means we must use a full text keyboard instead. This is unfortunately not an ideal experience, but it is the only way to allow a user to enter all possible numbers.
+
+<div style={{display: 'flex', flexDirection: 'row', justifyContent: 'space-around', flexWrap: 'wrap'}}>
+  <figure style={{width: '200px'}}>
+    <img src={numberfieldIphoneInteger} style={{margin: 0, width: '100%'}} alt="Iphone screenshot of example NumberField inputMode=numeric" />
+    <figcaption>With inputMode="numeric", iOS does not include a minus key or a decimal point.</figcaption>
```

Comment:
these captions have the browser default font and not the spectrum clean, also applies elsewhere too in other pages like the color one